From c0a516b09316a8bb73c872d9da2f584e5ed5b112 Mon Sep 17 00:00:00 2001
From: Henry Hiles <henry@henryhiles.com>
Date: Mon, 21 Jul 2025 21:27:10 -0400
Subject: [PATCH] Fix update script for nix

---
 bin/update.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/update.sh b/bin/update.sh
index 5058f33f16..41361d9481 100755
--- a/bin/update.sh
+++ b/bin/update.sh
@@ -19,7 +19,7 @@
  +-----------------------------------------------------------------------+
 */
 
-define('INSTALL_PATH', realpath(__DIR__ . '/..') . '/');
+define('INSTALL_PATH', dirname(dirname($argv[0])).'/');
 
 require_once INSTALL_PATH . 'program/include/clisetup.php';
 
